corehq/apps/cloudcare/views.py

```diff
@@ -554,3 +556,14 @@ def _message_to_tag_value(message, allowed_chars=string.ascii_lowercase + string
     message_tag = ''.join((c if c in allowed_chars else ' ') for c in message_tag.lower())
     message_tag = '_'.join(re.split(r' +', message_tag)[:4])
     return message_tag[:59]
+
+
+def _message_to_sentry_thread_topic(message):
+    """
+    >>> _message_to_sentry_thread_topic(
+    ... 'EntityScreen EntityScreen [Detail=org.commcare.suite.model.Detail@1f984e3c, '
+    ... 'selection=null] could not select case 8854f3583f6f46e69af59fddc9f9428d. '
+    ... 'If this error persists please report a bug to CommCareHQ.')
+    'EntityScreen EntityScreen [Detail=org.commcare.suite.model.Detail@[...], selection=null] could not select case [...]. If this error persists please report a bug to CommCareHQ.'
+    """
+    return re.sub(r'[a-f-0-9]{8,}', '[...]', message)
```

Comment:
Nitpick: what would you think of backslashing the middle hyphen (which admittedly might trigger an unnecessary backslash warning) or putting it at the end so it's more obvious that it's meant to match hyphens instead of indicate a character range?

Reply:
Sure, makes sense